distributed/stealing.py

```diff
@@ -85,7 +84,6 @@ def put_key_in_stealable(self, ts):
         ws = ts.processing_on
         worker = ws.address
         cost_multiplier, level = self.steal_time_ratio(ts)
-        self.log(("add-stealable", ts.key, worker, level))
```

Comment:
Doing the same thing for `remove_key_from_stealable` ( https://github.com/dask/distributed/pull/4609 )